documentation/staging/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -29,9 +29,9 @@ Deploy the operator and ensure that it is monitoring the desired namespace for y
 
 Model in Image requires an image with a WebLogic Server installation.
 
-- You can start with a WebLogic Server 12.2.1.3 or later Oracle Container Registry pre-built base image such as `container-registry.oracle.com/middleware/weblogic:12.2.1.3` for WLS domains or `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3` for JRF domains. For an example of this approach for both WLS and JRF domains, see the [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}) sample. For detailed instructions on how to log in to the Oracle Container Registry and accept the license agreement for an image (required to allow pulling an Oracle Container Registry image), see this [document]({{< relref "/userguide/base-images/_index.md#obtain-images-from-the-oracle-container-registry" >}}).
+- You can start with a WebLogic Server 12.2.1.3 or later Oracle Container Registry pre-built base image such as `container-registry.oracle.com/middleware/weblogic:12.2.1.3` for WLS domains or `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3` for JRF domains. For an example of this approach for both WLS and JRF domains, see the [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}) sample. For detailed instructions on how to log in to the Oracle Container Registry and accept the license agreement for an image (required to allow pulling an Oracle Container Registry image), see this [document]({{< relref "/userguide/base-images/ocr-images#obtain-images-from-the-oracle-container-registry" >}}).
```

Comment:
The idea of starting with a GA image as the base and then applying PSUs and other patches on top of them will result in a bloated image.  This is due to the fact that the image tool cannot optimize out content that lives in the base image.  Customers that want to apply their own patches should really start with the GA installers and patches that they want to minimize the size of the resulting image.

Reply:
This particular piece of doc is not instructing which image to patch with PSU, but which image to add models to. It's assuming a base GA image in a developer env -- but this is not a "best practice" except for demos, etc, so Rosemary is trying to find all GA usage locations and add a warning about using GA images per Request from Derek, Monica, and I...  

In this particular line of doc I suspect we can simply remove the example image name entirely.

Reply:
**First**, let me clarify: I have **not** been asked to try find all GA usage locations and add a warning about using GA images per request from Derek and Monica. Yes, Tom mentioned it to me, but that was not accomplished in this PR. I edited the x-ref in this file. I will gladly add the GA warning to it, however, I will require further instruction from Monica and/or Derek with regard to exactly what needs to be done for "all GA usage locations." 

**Second**, Robert's point is not the same one being made by the GA Warning message: "WebLogic Server General Availability images and Fusion Middleware Infrastructure General Availability images on OCR are updated quarterly to include the latest security patches for Oracle Linux and Oracle Java, but do not include the latest security patches for WebLogic Server or Fusion Middleware Infrastructure. Oracle strongly recommends using images with all the latest security patches, such as Critical Patch Updates images provided quarterly on OCR or custom generated images using the WebLogic Image Tool (WIT) with the --recommendedPatches option. See [Ensure you are using recently patched images](http://localhost:1313/weblogic-kubernetes-operator/userguide/base-images/ocr-images/#ensure-you-are-using-recently-patched-images)." 

@robertpatrick @tbarnes-us Which change do you want me to make to this file, or should I do both?

Reply:
I would like this documentation to be in line with the following documentation which has been added to FMW docs here https://docs.oracle.com/en/middleware/fusion-middleware/12.2.1.4/opatc/obtaining-creating-and-updating-oracle-fusion-middleware-images-patches.html.  This guidance includes the comments made by Robert about the use of WIT "create" when  applying PSUs, SPUs, and bundle patches because the size of the image becomes large.

Rosemary the guidance that Derek and I provided and discussed for the WIT documentation still applies there and is consistent to what we should document here.

Let's have a meeting and discuss what we should document in the WKO userguide/managing-domains

Reply:
Done.